src/api/entityResolution/types.ts

```diff
@@ -279,3 +279,59 @@ export type IngestionStatus =
        */
       timeLeftSeconds?: number;
     };
+
+/**
+ *  Informations about the entity resolution license, across all data-sources.
+ */
+export interface EntityResolutionLicenseInfo {
+  /**
+   * The number of records ingested.
+   */
+  recordsIngested: number;
+  /**
+   * The number of records available in the license.
+   */
+  recordsAvailable: number;
+}
```

Comment:
We may later add other properties to this interface. Typically, the license expiration date.